### DISCUSSION TITLE
Multi core registration

### DISCUSSION THREAD

**Discussion by JonathanAkaJD** [masterking32/WoWSimpleRegistration#93] [Q&A]
Would it be complicated to integrate registration on multiple cores at once? For example, say you have an azerothcore and a skyfire core or an azerothcore and a mangos core. With authentication databases not being cross compatible, it's frustrating managing multiple registration scripts for multiple cores and explaining to users a need to reregister.

**Comment by masterking32** [comment]
I’m afraid there may be some confusion, but let me clarify. If you’re running a TrinityCore server and wish to migrate to AzerothCore without losing any accounts, it’s important to understand the following: Previously, around 3-4 years ago, both core servers used SHA1 for password encryption. However, this standard has since shifted. Assuming your initial core is outdated, you’ll first want to create a backup and then update it to the latest TrinityCore version. This update will automatically convert the password encryption to the newer SRP6 method.

Afterward, you can extract the necessary fields from your database and integrate them into the new core. It’s critical to ensure that both cores use the same encryption method; for instance, if your TrinityCore data utilizes SRP6, you cannot revert it back to SHA1 encryption.

Unfortunately, I don’t have a script available to facilitate this process, so you will need to perform these tasks manually.

You need to ask about this situation in your core community.

-----------------------------------------

Additionally, if you have two servers for authentication and maintain two separate authentication databases, you’ll need to establish two distinct MySQL connections for each database. When implementing account creation, you will need to either create an additional function or update the existing registration process to support multiple authentication sources, enabling the new account data to be stored in both databases simultaneously.

**Reply by JonathanAkaJD** [reply]
Hmm, I think there's been some confusion. I don't mean migration. Let me give a real world example. I have a skyfire core I maintain specifically for a mop(mist of pandaria) while I also have a Azerothcore for stable wrath of the lich king server. With the script as is, it displays online players for multiple cores, but registration across separate cores isn't possible. It would be nice if when a user signs up, it makes an account on both cores. So long story short, the feature I'm asking is much like how I can display two different cores, would it be difficult to integrate cross core registration?

**Reply by masterking32** [reply]
There is another part in the answer:

> Additionally, if you have two servers for authentication and maintain two separate authentication databases, you’ll need to establish two distinct MySQL connections for each database. When implementing account creation, you will need to either create an additional function or update the existing registration process to support multiple authentication sources, enabling the new account data to be stored in both databases simultaneously.
